src-graphql/client.js

```diff
@@ -7,6 +7,29 @@ import productConnectionQuery from './product-connection-query';
 import collectionQuery from './collection-query';
 import collectionConnectionQuery from './collection-connection-query';
 
+function fetchAll(paginatedModels, client) {
```

Comment:
Sorry, I'd still differentiate the name here, and call it something like `fetchAllPages`, so it's clear it's not just the index, it's literally everything.